Open Judge System/Services/OJS.Services/Business/Contests/Models/BaseContestBusinessModel.cs

```diff
@@ -35,14 +35,14 @@ public abstract class BaseContestBusinessModel
         public int ActualWorkers { get; set; }
 
         [Range(1, 100)]
-        public int PreviousContestExpectedProblems { get; set; }
+        public int PreviousContestExpectedProblems { get; set; } = 1;
 
         [Range(1, int.MaxValue)]
         public int CurrentContestId { get; set; }
 
         public int? PreviousContestId { get; set; }
 
-        public int PreviousAverageProblemRunTimeInSeconds { get; set; }
+        public int PreviousAverageProblemRunTimeInSeconds { get; set; } = 1;
```

Comment:
Why 1s? Better set 0s or make them nullable, so it is clear this value is not retrieved from anywhere